pkg/plugins/manager/manager_test.go

```diff
@@ -106,23 +131,66 @@ func TestPluginManager_Init(t *testing.T) {
 	})
 
 	t.Run("With external back-end plugin with valid v2 signature", func(t *testing.T) {
+		pluginsDir := "testdata/valid-v2-signature"
+		pluginFolder := pluginsDir + "/plugin"
```

Comment:
Nit, but these are better represented as consts:
```suggestion
		const pluginsDir = "testdata/valid-v2-signature"
		const pluginFolder = pluginsDir + "/plugin"
```